### PR TITLE
suggested changes to solve an issue with floating point number formatting

### PR DESCRIPTION
**Pull request by nevrome**
As discussed in the chat after Taylor's comments.

### CODE REVIEW

File: R/dataprep_table_formatting.R [hunk 1]
```diff
@@ -43,7 +43,7 @@ format_as_update_existing <- function(sidora_table) {
   # as displayed (particularly to prevent datetime saving with TZ info)
   result %>% 
     dplyr::rename_with(make_column_mandatory, valid_cols_table_clean_mandatory) %>%
-    dplyr::mutate(dplyr::across(tidyselect::everything(), as.character))
+    dplyr::mutate(dplyr::across(tidyselect::everything(), format, trim = TRUE))
```
Comment (nevrome) [inline R/dataprep_table_formatting.R:46]: I'm not entirely sure if that is stable across all data types

File: R/dataprep_table_formatting.R [hunk 2]
```diff
@@ -76,7 +76,7 @@ fix_logical_update_existing <- function(sidora_table){
   
   if ("sample.Ethically_culturally_sensitive" %in% names(sidora_table)) {
     result <- sidora_table %>% 
-      dplyr::mutate(dplyr::across(dplyr::contains(c("sample", ".Ethically_culturally_sensitive")), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
+      dplyr::mutate(dplyr::across(dplyr::contains(c("sample.Ethically_culturally_sensitive")), function(x) gsub(F, "No", x) %>% gsub(T, "Yes", .)))
```
Comment (nevrome) [inline R/dataprep_table_formatting.R:79]: Why was that separated in the first place? With `c("sample", ".Ethically_culturally_sensitive")`, every column of TAB_Sample was already transformed to character here.

File: R/dataprep_table_formatting.R (hunk 1 above)
Comment (jfy133) [reply R/dataprep_table_formatting.R:46]: Can try and wait for the explosions

File: R/dataprep_table_formatting.R (hunk 2 above)
Comment (jfy133) [reply R/dataprep_table_formatting.R:79]: Not sure now... indeed doesn't make sense.

**Review by jfy133 — APPROVED** [review]

**Pull request merged**